code-studio/src/js/initApp/project.js

```diff
@@ -231,7 +233,7 @@ var projects = module.exports = {
    * here whether we're an admin, and depend on dashboard getting this right.
    */
   showAdmin: function() {
-    dashboard.admin.showProjectAdmin();
+    showProjectAdmin();
```

Comment:
Could remove the `showAdmin()` function altogether - it looks like it's only called once, and from inside this file.